database/tables.py

```diff
@@ -4,9 +4,9 @@
 
 import logging
 
-from typing import List, Optional, Tuple
+from typing import List, Optional, Tuple, Dict
 from datetime import datetime, timedelta
-from pytz import timezone
+from pytz import timezone, BaseTzInfo
```

Comment:
`timezone` is unused
```suggestion
from pytz import BaseTzInfo
```